NEWS.md

```diff
@@ -1,5 +1,9 @@
 # rlang (development version)
 
+* New `env_coalesce()` function to copy bindings from one environment
```

Comment:
Another option would be `env_modify()` by analogy to `modifyList()`. I don't mind `coalesce()` either.

Reply:
hmm good point but I think this would suggest a similar UI to `call_modify()`.